src/gwt/src/org/rstudio/core/client/VirtualConsole.java

```diff
@@ -1173,7 +1209,7 @@ private void setText(String text)
          }
          else
          {
-            element.setInnerText(truncate(text));
+            element.setInnerText(text);
```

Comment:
I need to figure out a more appropriate place to truncate, since as-is we're truncating every time new text is added to a truncate-able span.